conformance/local_symtab.ion

```diff
@@ -0,0 +1,71 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+// Non-structs pass through
+(ion_1_x
+  (then (toplevel first $ion_symbol_table::[] second)
+        (produces first $ion_symbol_table::[] second))
+  (then (toplevel first $ion_symbol_table::() second)
+        (produces first $ion_symbol_table::() second)))
+
+// IST structs are elided from app view
+(ion_1_x
+  (then (toplevel first $ion_symbol_table::{} $ion_symbol_table::null.struct second)
+        (produces first                                                      second))
+  // ...but only at top-level
+  (then (toplevel [first, $ion_symbol_table::{}, $ion_symbol_table::null.struct, second])
+        (produces [first, $ion_symbol_table::{}, $ion_symbol_table::null.struct, second])))
+
+
+// Basic symbol expansion
+(ion_1_0 (toplevel $ion_symbol_table::{symbols:["a"]})
```

Comment:
Suggestion—we should make sure that `(toplevel $3::{ $7: ["a"] } )` here would also produce the same thing.

Reply:
Good idea, will do.